src/braket/aws/aws_quantum_job.py

```diff
@@ -165,6 +165,7 @@ def create(
         Raises:
             ValueError: Raises ValueError if the parameters are not valid.
         """
+
```

Comment:
nit: no opinion on the whitespace, but this change pollutes the diff by adding an extra file. If you want to be extra, you can fix this (you can do this inline on github)

Reply:
Oh sorry, I didn't notice because the commits were not squashed. How do I fix this inline?

Reply:
Looks like you figured this out-- for anyone seeing this, you can add your own review with a code change suggestion and then select "commit suggestion"

Reply:
Ok thanks, I was able to edit the file directly in Github and commit it.